src/main/java/tech/jhipster/lite/cli/command/infrastructure/primary/JHLiteCommandsFactory.java

```diff
@@ -1,20 +1,26 @@
 package tech.jhipster.lite.cli.command.infrastructure.primary;
 
 import java.util.List;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 import picocli.CommandLine.Model.CommandSpec;
 
 @Component
 class JHLiteCommandsFactory {
 
   private final List<JHLiteCommand> jhliteCommands;
+  private final String version;
 
-  public JHLiteCommandsFactory(List<JHLiteCommand> jhliteCommands) {
+  public JHLiteCommandsFactory(List<JHLiteCommand> jhliteCommands, @Value("${project.version:0.0.1-SNAPSHOT}") String version) {
```

Comment:
Do we really need a default value?
```suggestion
  public JHLiteCommandsFactory(List<JHLiteCommand> jhliteCommands, @Value("${project.version}") String version) {
```

Reply:
Done :)